composer.json

```diff
@@ -27,7 +27,8 @@
     "psr/http-client-implementation": "^1.0",
     "psr/http-message": "^1.0",
     "tightenco/collect": "^7.0|^8.0",
-    "beberlei/assert": "^3.2"
+    "beberlei/assert": "^3.2",
+    "symfony/symfony": "5.4.16"
```

Comment:
Why are you pulling symfony?

Reply:
There was an issue with code style and test pass on php 8 but fails on php 7.4, and it had to do with an update on symfony. Kindly find full discussions here:

https://themailergroup.slack.com/archives/C02657BK2MU/p1671610967361939

Consideration was to lock down symfony to a specific version that works with php 7.4